packages/next/shared/lib/loadable.js

```diff
@@ -158,11 +162,20 @@ function createLoadableComponent(loadFn, options) {
   }
 
   function LazyImpl(props, ref) {
+    useLoadableModule()
+
+    const context = React.useContext(LoadableContext)
+    if (context && Array.isArray(opts.modules)) {
+      opts.modules.forEach((moduleName) => {
+        context(moduleName)
+      })
+    }
```

Comment:
Is this needed if `useLoadableModule()` is already iterating and calling the modules?